### PR TITLE
REPL

### PR DESCRIPTION
**Pull request by coco-sha**

### CODE REVIEW

File: packages/polar/src/builtin-tasks/repl.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    } else {
+      console.error(e);
+      callback(null);
+    }
+  }
+}
+
+async function startConsole (runtimeEnv: PolarRuntimeEnvironment): Promise<void> {
+  await new Promise<void>((resolve, reject) => {
+    console.log("★", chalk.blueBright(" Welcome to polar console"), "★");
```
Comment (coco-sha) [inline packages/polar/src/builtin-tasks/repl.ts:29]:
```suggestion
    console.log("★", chalk.blueBright(" Welcome to polar REPL"), "★");
```

File: packages/polar/src/builtin-tasks/repl.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    });
+  });
+}
+
+export default function (): void {
+  task(TASK_REPL, "Opens polar console")
+    .addFlag("noCompile", "Don't compile before running this task")
+    .setAction(
+      async (
+        { noCompile }: { noCompile: boolean },
```
Comment (udit-gulati) [inline packages/polar/src/builtin-tasks/repl.ts:53]: Is `noCompile` being used here?

File: packages/polar/src/internal/util/repl.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+export function isNodeCalledWithoutAScript (): boolean {
+  const script = process.argv[1];
+  return script === undefined || script.trim() === "";
+}
+
+/**
+ * Starting at node 10, proxies are shown in the console by default, instead
+ * of actually inspecting them. This makes all our lazy loading efforts wicked,
+ * so we disable it ni buidler/register.
```
Comment (udit-gulati) [inline packages/polar/src/internal/util/repl.ts:9]: typo

**Review by udit-gulati — CHANGES_REQUESTED** [review]

File: packages/polar/src/internal/util/repl.ts (hunk 3 above)
Comment (coco-sha) [inline packages/polar/src/internal/util/repl.ts:9]:
```suggestion
 * so we disable it in register.
```

File: packages/polar/src/builtin-tasks/repl.ts (hunk 2 above)
Comment (coco-sha) [reply packages/polar/src/builtin-tasks/repl.ts]: removed

**Review by udit-gulati — APPROVED** [review]

**Pull request merged**